### ISSUE TITLE
Недоступно обновление списка в LuCI

### ISSUE THREAD

**Issue by Yoti** [remittor/zapret-openwrt#11]
Используя Zapret v67-20241030 на OpenWrt 23.05.5 имеем такую картину:
`Blacklist update mode: user entries only` (и нет настройки смены режима)
`Update blacklist [Update]` (кнопка неактивна при обоих `MODE_FILTER`)
Это пока мёртвый задел на будущее или я упустил какие-то нюансы?

**Comment by remittor** [comment]
Это задел на будущее. На текущий момент все списки нужно заполнять вручную.

**Comment by Yoti** [comment]
Переоткрою issue, так как возник вопрос по схожей тематике. В `/etc/crontabs/root` не появилось записи на обновление, в отличие от ручной установки zapret. Так задумано или это баг, @remittor?

**Comment by remittor** [comment]
Оригинальный `install_easy.sh` не допустимо использовать, т.к. используется механизм установки предусмотренный OpenWrt (opkg).

**Comment by Yoti** [comment]
Да, на запись в cron всё относится к процессу последующей работы, хоть и создаётся в процессе установки.

**Comment by remittor** [comment]
> запись в cron всё же относится к процессу последующей работы

Так это используется для авто обновления списков:
https://github.com/bol-van/zapret/blob/db0caf7d5805b66f1e1d7170f31ce1146016bb37/common/installer.sh#L290

У меня пока такое не поддерживается.

**Comment by Yoti** [comment]
А чего тут поддерживать? Оно само работает раз в пару дней. Пока нет настроек, лучше уж чтобы работало хоть что-то, чем вообще ничего.
